Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+                  </div>
+                    <button><a class="boton2">See Project</a></button> 
+                  </div>
+
+    </section>
+
+  <section class="page3">
+    <h1 class="about">About me</h1>
+    <p class="parrafoAM">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <button><a class="boton3">Get My Resume</a></button> 
+    <img class="vector2"src="./images/Vector.svg">
+    <div class="skills">
+      <ul>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-diamond"></i>     Languaje</li>
+          <li class="smallskill">JavaScript</li>
+          <li class="smallskill">Ruby</li> 
+          <li class="smallskill">Html</li>
+          <li class="smallskill">Css</h4></li> 
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-square"></i>     Frameworks</li>
+           <li class="smallskill">Bootstrap</li>
+           <li class="smallskill">Ruby_on_Rails</li> 
+           <li class="smallskill">RSPec</li>  
+           <li class="smallskill">CapyBara</li>
+           <li class="smallskill">Selenium</li>
+        <li class="bigskill"><i class="fa-sharp fa-solid fa-circle"></i>     Skills</li>
+           <li class="smallskill">Codekit</li>
+           <li class="smallskill">Github</li> 
+           <li class="smallskill">Codepen</li>
+           <li class="smallskill">Gitlab</li>
+           <li class="smallskill">Terminal</li>  
+      </ul>
+    </div>
+  </section>
+  <section class="page4">
+    <p class="pfinal">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+  <form class="cuadrollenado"> 
+    <p class="fill">Full name</p>
+  <img class="fillbar"src="./images/fillbar.svg">
+  <p class="fill">Email address</p> 
+  <img class="fillbar"src="./images/fillbar.svg">
+  <textarea class="comments">Enter text here</textarea>
+</form>
+  <button><a class="boton4">Get in touch</a></button>
+  <div class="final_icons">
+    <ul>
+    <li<a href="https://github.com/Zuz4n/"><img class="github"src="./images/Github.svg"></a></li>
+    <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="linkedin"src="./images/Linkedin.svg"></a></li>
+    <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="newent"src="./images/new.svg"></a></li>
+    <li<a href="https://twitter.com/SuschenSu"><img class="twitter"src="./images/Twitter.svg"></a></li>
+    <li<a href="siuxmicroverse@gmail.com"><img class="mail"src="./images/Mail.svg"></a></li>
+  </ul>
+  </div>
+  <img class="fb"src="./images/finalbar.svg">
+</section>
```

Comment:
- [Optional] You don't need this section just yet. There will be a project that will require ou to implement this section. Consider stashing it somewhere for now.